pbm/backup/backup.go

```diff
@@ -183,6 +183,14 @@ func (b *Backup) run(bcp pbm.BackupCmd) (err error) {
 		return errors.Wrap(err, "waiting for start")
 	}
 
+	// A fallback. If there are some leftovers and main node
+	// hasn't cleaned up it yet we should stop and notify user
+	// since backup would be unrestorable.
+	err = b.node.EnsureNoTMPcoll()
+	if err != nil {
+		return errors.Wrap(err, "EnsureNoTMPcoll")
```

Comment:
```suggestion
		return errors.Wrap(err, "Error-exiting from Backup run() because processing check EnsureNoTMPcoll() failed. See preceding messages in log for collection names.")
```

Reply:
The error here will contain collections name. Now it looks like:
```
backup: EnsureNoTMPcoll: pbm tmp collections exists. It will lead to failed restore. Drop collections: admin.pbmRRoles, admin.pbmRUsers
```

looking at it now I reckon it needs some rephrasing. I suggest:
```
backup: EnsureNoTMPcoll: PBM temporary collections exists. It will lead to a failed restore. Please, drop next collections manually: admin.pbmRRoles, admin.pbmRUsers
```

I don't quite in favour of long phrasing in logs. E.g. `Error-exiting from Backup run() because processing check EnsureNoTMPcoll() failed` - anyway gives nothing valuable to the user, and for programmer `EnsureNoTMPcoll ` is enough to identify the place in the code and it makes less visual noise in the log.